content/00.front-matter.md

```diff
@@ -1,6 +1,5 @@
-# Manubot Rootstock: Manuscript Title
 
-## Authors
+# Authors
```

Comment:
Yeah. It will actually work better with them at level 2 under the title. It will allow you to have supplementary content as a level 1 header at end, with level 2's under it. That way, in the pdf navigation, the top levels are broken down into the major divisions of the manuscript.